docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,26 @@
+# Troubleshooting
+
+## Known Problems + Solutions
+
+1) If you uninstall Karpenter after creating some nodes with Karpenter, you will be left with nodes that have finalizers that need to be patched out for the node object to be deleted.
+
+To do this, you can do either of the following:
+- `kubectl edit node <node_name>` and remove the line that says `karpenter.sh/termination` in the finalizers field.
+- Run the following script that gets all nodes with the finalizer and removes all the finalizers from those nodes. NOTE: this will remove ALL finalizers from nodes with the karpenter finalizer.
+```{bash}
+kubectl get nodes -ojsonpath='{range .items[*].metadata}{@.name}:{@.finalizers}{"\n"}' | grep "karpenter.sh/termination" | cut -d ':' -f 1 | xargs kubectl patch node --type='json' -p='[{"op": "remove", "path": "/metadata/finalizers"}]'
```

Comment:
Is json the default patch type?

Reply:
Nope. It looks the cleanest to read in my opinion though.